src/routes/(app)/admin[[hash=admin_hash]]/nostr/+page.svelte

```diff
@@ -80,13 +80,22 @@
 </form>
 
 <h2 class="text-2xl">Relays</h2>
-
-<ul>
-	{#each data.nostrRelays as relay}
-		<li>{relay}</li>
-	{/each}
-</ul>
-
+<form action="?/addRelay" method="post" class="flex flex-col gap-4">
+	<ul>
+		{#each data.nostrRelays as relay}
+			<li>{relay}</li>
+			<input type="hidden" name="relays" value={relay} />
+		{/each}
+	</ul>
+	<label class="form-label">
+		Relay
+		<input class="form-input" type="text" name="relays" placeholder="wss://new.relay.url" />
+	</label>
+	<div class="flex justify-between gap-2">
+		<button class="btn btn-black self-start" type="submit">Add relay</button>
+		<input type="submit" value="delete" class="btn btn-red self-end" formaction="?/deleteRelay" />
```

Comment:
Instead of a new endpoint, you can add a trashcan icon button for each relay in the `<li>` element, it will remove the relay from the UI and the form data afterwards when sent.

It also allows the user to remove relays at the beginning, middle, etc.